.github/workflows/quick-jobs.yml

```diff
@@ -145,6 +145,7 @@ jobs:
         run: make doc/buildinfo-fields-reference.rst
       - name: Cache dependencies
         uses: actions/cache/save@v4
+        # 'always()' here overrides job cancellation
```

Comment:
Looks clumsy that the same thing is phrased differently than in bootstrap.yml

Reply:
Still sorting out how best to put it, and addition of above about why it's wanted. These will all change, probably by explaining it in one place and referencing it in the others.